### PR TITLE
fix: create event loop manually in case of running within non-main thread

### PR DESCRIPTION
**Pull request by Mattwmaster58**
I believe this fixes what #350 is referring to.

### CODE REVIEW

**Comment by Mattwmaster58** [comment]
This has issues when it comes to Linux + Python 3.7 that will have to be addressed.

File: playwright/main.py [hunk 1]
```diff
@@ -59,9 +60,13 @@ async def run_driver_async() -> Connection:
 
 
 def run_driver() -> Connection:
-    loop = asyncio.get_event_loop()
-    if loop.is_running():
-        raise Error("Can only run one Playwright at a time.")
+    # if we're running in a thread, get_event_loop won't create an event loop for us, we'll need to make it ourselves
+    if not isinstance(threading.current_thread(), threading._MainThread):  # type: ignore
```
Comment (pavelfeldman) [inline playwright/main.py:64]: We should probably reuse existing loop, regardless of the thread. I have a code health patch that resolves it as a driver-by: https://github.com/microsoft/playwright-python/pull/354/files

Comment (Mattwmaster58) [reply playwright/main.py:64]: I wasn't aware. Feel free to open if this is still relevant.

Comment (Mattwmaster58) [reply playwright/main.py:64]: This is basically checking for one of the conditions which will cause `RuntimeError` to be raised an avoiding it prematurely.

File: playwright/main.py [hunk 2]
```diff
@@ -112,9 +117,7 @@ async def __aexit__(self, *args: Any) -> None:
 
 
 if sys.platform == "win32":
-    # Use ProactorEventLoop in 3.7, which is default in 3.8
-    loop = asyncio.ProactorEventLoop()
-    asyncio.set_event_loop(loop)
+    asyncio.set_event_loop_policy(asyncio.WindowsProactorEventLoopPolicy())
```
Comment (pavelfeldman) [inline playwright/main.py:120]: Now I know why you did this change @Mattwmaster58 !

**Pull request closed without merging**